contracts/external/cw-payroll-factory/schema/cw-payroll-factory.json

```diff
@@ -276,7 +276,7 @@
             ]
           },
           "start_time": {
-            "description": "The time to start vesting, or None to start vesting when the contract is instantiated.",
+            "description": "The time to start vesting, or None to start vesting when the contract is instantiated. This may be a time in the past, though it is checked that `start_time + vesting_duration_seconds > now` as a vest that completes instantly is equivalent to a regular fund transfer, but with a lot of extra code.",
```

Comment:
> `start_time` may be in the past, though the contract checks that `start_time + vesting_duration_seconds > now`. Otherwise, this would amount to a regular fund transfer.

Reply:
thank you! much better.